doc/sphinx-guides/source/developers/dev-environment.rst

```diff
@@ -57,6 +57,10 @@ Launch Netbeans and click "File" and then "Open Project". Navigate to where you
 
 If you installed Maven instead of Netbeans, run ``mvn package``.
 
+NOTE: Do you use a locale different than ``en_US.UTF-8`` and a timezone different to ``UTC`` on your development machine?
```

Comment:
This is a little confusing to me because the core development team at @IQSS is not in the "UTC" timezone. We're in Boston (well, across the river in Cambridge) so it's Eastern Time Zone which I guess is either "EST" or "EDT" according to https://en.wikipedia.org/wiki/Eastern_Time_Zone . Maybe this could say something like "we set our tests to UTC" for testing?